src/codegen/compile_classical.ml

```diff
@@ -5059,6 +5059,8 @@ module IC = struct
 
   (* IC-specific stuff: System imports, databufs etc. *)
 
+  let i = I32Type
```

Comment:
What (semantic) difference do you want to emphasise with this?

Reply:
My idea was to mimic the `I` type from the [ic interface spec](https://internetcomputer.org/docs/references/ic-interface-spec#ic-sign_with_ecdsa). That way it is easier to:
- maintain both `compile_classical.ml` and the `compile_enhanced.ml` -- the code in both is the same, just the value of `i` changes Int32/Int64
- understand the meaning that this is more of a pointer and that it refers to the same concept as in the interface spec

Reply:
If we agree that this is a good idea, then I'll put a comment describing that next to the definition of `i`